### PR TITLE
Fix a bug where mobs can become stuck when transitioning to the standing state

### LINKED ISSUE #26068: Lube can sling you inside walls.
## Description
Lube has enough force to put you inside walls. Killing people unlucky enough to step in the wrong place.
In the screenshot two engineers walked in to engineering and got stuck in wall above table and died. RIP.

**Reproduction**
See image. Short runway, maybe the table cell charger affects but have seen other times people in walls.

**Screenshots**
image

**Additional context**
Feature not bug?

### PR DESCRIPTION
**Pull request by Brandon-Huu**
Reopening of #24122 
Resolves #26068

### CODE REVIEW

File: Content.Shared/Standing/StandingStateSystem.cs [hunk 1]
```diff
@@ -67,12 +77,22 @@ public bool IsDown(EntityUid uid, StandingStateComponent? standingState = null)
...
+                // Create a temporary sensor to continue registering contacts in the disabled collision layer
+                _fixtureSystem.TryCreateFixture(
+                    uid,
+                    new PhysShapeCircle(0.1f),
+                    StandingSensorFixtureName,
+                    collisionLayer: (int) CollisionGroup.None,
+                    collisionMask: StandingCollisionLayer,
+                    hard: false,
+                    manager: fixtureComponent);
             }
```
Comment (metalgearsloth) [inline Content.Shared/Standing/StandingStateSystem.cs:96]: Why are dummy fixtures getting made.

Comment (Brandon-Huu) [reply Content.Shared/Standing/StandingStateSystem.cs:96]: Similar to

https://github.com/space-wizards/space-station-14/blob/85e0a5328a28b02fc29575648cfa7421f5bfb410/Content.Shared/Climbing/Systems/ClimbSystem.cs#L305-L338

It's used for detecting when a player is no longer inside an object they were slipped / dragged into

**Comment by Brandon-Huu** [comment]
People have been using this bug to phase through windoors

File: Content.Shared/Standing/StandingStateSystem.cs (hunk 1 above)
Comment (metalgearsloth) [reply Content.Shared/Standing/StandingStateSystem.cs:96]: What's wrong with updating the existing fixture? This isn't going to have the correct bounds or ID handling.

**Comment by metalgearsloth** [comment]
Clumsy part got fixed but review outstanding and idk if this does anything anymore.

**Pull request closed without merging**